samples/instrumentation-quickstart/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM python:3.12
+WORKDIR /usr/src/app
+COPY . .
+RUN python -m venv /venv && \
+    /venv/bin/pip install -r requirements.txt
+CMD bash -c "source /venv/bin/activate && flask run --host 0.0.0.0 --port 8080 2>&1 | tee /var/log/app.log"
```

Comment:
I think for this sample it would be nice to have a more realistic productionized app. Typically people use Gunicorn as a WSGI server, see https://flask.palletsprojects.com/en/3.0.x/deploying/.

Feel free to add a TODO or address in another PR

Reply:
added a TODO